apm-server.yml

```diff
@@ -384,13 +379,11 @@ output.elasticsearch:
   #    when.contains:
   #      processor.event: "onboarding"
 
-  # A pipeline is a definition of processors applied to documents when writing them to Elasticsearch.
-  # APM Server comes with a default pipeline definition, located at `ingest/pipeline/definition.json`.
-  # Pipelines are disabled by default. To make use of them you have to:
-  # (1) ensure pipelines are registered in Elasticsearch, see `apm-server.register.ingest.pipeline`
-  # (2) enable the following:
-  #pipelines:
-  #- pipeline: "apm"
+  # A pipeline is a definition of processors applied to documents when ingesting them to Elasticsearch.
+  # APM Server comes with a default pipeline definition, located at `ingest/pipeline/definition.json`, which is
+  # loaded to Elasticsearch by default (see `apm-server.register.ingest.pipeline`).
+  # APM pipeline is enabled by default. For disabling set `pipeline: _none`.
+  #pipeline: "apm"
```

Comment:
For my knowledge, this can still be an array, right? If I wanted to register multiple pipelines

Reply:
Multiple pipelines would be registered in `pipelines`, since we only have one I changed the default to use `pipeline`.